### PR TITLE
`unreleased-commits` - Fix icon alignment

### PR DESCRIPTION
**Pull request by kidonng**
Small cosmetic tweak

## Test URLs

https://github.com/refined-github/refined-github

## Screenshot

| Before | After |
|--------|--------|
| Screenshot 2024-06-30 at 1 58 47 PM | Screenshot 2024-06-30 at 2 04 30 PM |

### CODE REVIEW

File: source/features/unreleased-commits.tsx [hunk 1]
```diff
@@ -91,8 +91,8 @@ async function createLink(
 			href={buildRepoURL('compare', `${latestTag}...${await getDefaultBranch()}`)}
 			aria-label={label}
 		>
-			<TagIcon className="v-align-middle"/>
-			{aheadBy === undeterminableAheadBy || <sup className="ml-n2"> +{aheadBy}</sup>}
```
Comment (fregante) [inline source/features/unreleased-commits.tsx:95]: n2 was intentionally to bring it closer together, it's not a label but it's "part of the icon"

Comment (kidonng) [reply source/features/unreleased-commits.tsx:95]: Only remove `v-align-middle`:

Screenshot 2024-06-30 at 2 20 55 PM

Ok-ish but in my opinion we should have a little space here.

**Review by fregante — CHANGES_REQUESTED** [review]
I think it’s fine as is. If you think of the + as part of the icon, it doesn’t look off center. The visual weight of the icon is somewhat balanced. In your last screenshot there’s a lot of space below the label and icon and it looks off balance.

**Comment by fregante** [comment]
Can you resend this?

- align + icon
- align tag icon only if count is missing

Because this does look misaligned:

Screenshot 15

**Pull request closed without merging**